doc/users/whats_new.rst

```diff
@@ -22,6 +22,27 @@ revision, see the :ref:`github-stats`.
 
       next_whats_new/*
 
+Improved default backend selection
+----------------------------------
+
+The default backend no longer must be set as part of the build
+process.  Instead, at run time, the builtin backends are tried in
+sequence until one of them imports.
+
+Headless linux servers (identified by the DISPLAY env not being defined)
+will not select a GUI backend.
+
+Cyclic colormaps
+----------------
+
+Two new colormaps named 'twilight' and 'twilight_shifted' have been added.
+These colormaps start and end on the same color, and have two
+symmetric halves with equal lightness, but diverging color. Since they
+wrap around, they are a good choice for cyclic data such as phase
+angles, compass directions, or time of day. Like *viridis*, *twilight* is
```

Comment:
Just a small suggestion...
Would it be better to mention *cividis* instead of *viridis* as the perceptually uniform and colorblind-friendly colormap? cividis explicitly avoids the green color since red-green is the most common colorblindness. 
https://nei.nih.gov/health/color_blindness/facts_about
https://arxiv.org/pdf/1712.01662.pdf

Reply:
That is a fair point, but I think viridis is better known at this point.  

The red-green issue was considered in the design of {viridis, magma, plasma, inferno} (which is why the first has green but no red and vise versa).

Reply:
I'll just put both in.